servers/rendering/rendering_server_default.cpp

```diff
@@ -42,26 +42,6 @@
 
 int RenderingServerDefault::changes = 0;
 
-/* BLACK BARS */
```

Comment:
Is this possible through a different API?
The UX of this method was maybe not the best, but I think being able to customize black bars is still something that some users enjoy, so we should re-add something for it eventually.

Reply:
It needs to be re-added again at some point, but with a different API and per window (not the case right now).